doc/buildAndProgram.md

```diff
@@ -65,8 +64,10 @@ DFU files are the files you'll need to install your build of InfiniTime using OT
 
 #### CMake command 
 
+ARM_NONE_EABI_TOOLCHAIN_PATH and NRF5_SDK_PATH must be specified (see above):
+
 ```
-cmake -DARM_NONE_EABI_TOOLCHAIN_PATH=... -DNRF5_SDK_PATH=...
+cmake -S "." -B "build" -DCMAKE_BUILD_TYPE=Release -DARM_NONE_EABI_TOOLCHAIN_PATH=... -DNRF5_SDK_PATH=.. -DBUILD_DFU=1 -DBUILD_RESOURCES=1 -DNRF_TARGET=nrf52
```

Comment:
`-DNRF_TARGET=nrf52` isn't needed, it's already specified in the cmake file. Release is also the default type, so I don't think that needs to be specified either (it's listed above, and I think people who need debug will know when they need it, you have to be pretty involved at that point)

As for DFU/resources, how about giving two example commands, one which builds neither (as current) and one which builds both, I think that way users can easily get the idea of what the options do